packages/swing-store-lmdb/src/lmdbSwingStore.js

```diff
@@ -2,18 +2,17 @@
 import fs from 'fs';
 import os from 'os';
 import path from 'path';
-import util from 'util';
-import Readlines from 'n-readlines';
 
 import lmdb from 'node-lmdb';
+import sqlite3 from 'better-sqlite3';
```

Comment:
it seems weird to have sqlite in the lmdbSwingStore file. will file this get renamed?

Reply:
I can take that out; it's a nod to ocap discipline where the `sqliteSwingStore` caller supplies the sqlite IO capability. That would make more sense if it were passed in all the way from the host application, or at least from the controller or something

The sqliteStreamStore does its own `import sqlite3` as a default, though; I can just use that.

Reply:
As our storage story has gotten more complicated, the LMDB store vs. simple store distinction has become increasingly weird and nonsensical.  Some kind of name refactoring is clearly called for, but not today.

Reply:
@FUDCo has proposed splitting these packages up. We use LMDB for the key-value portion of a "swing store", and flat files for the stream-store portion. This PR changes the stream-store portion to use sqlite, without changing the key-value store. A better-factored world would put each portion in its own package, and make a "swing store" parent package, or something.

I'm ok with the naming weirdness for today.